components/gif_picker/styles/icons.scss

```diff
@@ -0,0 +1,97 @@
+.ic {
+  display: inline-block;
+  background-repeat: no-repeat;
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+.ic-arrow-left {
+  width: .7rem;
+  height: 1.2rem;
+  background-size: .7rem 1.2rem;
+  background-image: url('../../static/icons/ic_arrow_greyblue_left.svg');
+  background-position: center;
+}
+
+.ic-reactions {
+  position: relative;
+  width: 2.2rem;
+  height: 2.2rem;
+  font-size: 2rem;
+}
+
+@keyframes play-loading {
+  from { background-position: 0; }
+  to { background-position: -10000px; }
+}
+
+.img-loading {
+  animation: play-loading 1.5s steps(50) infinite;
+  background-image: url('../../static/img_loading.png');
+  background-size: 10000px 200px;
+}
+
+.mattermost {
+  .ic-reactions {
+    color: #ffffff;
+  }
+}
+
+.ic-search {
+  width: 3.3rem;
+  height: 3.3rem;
+}
+
+.ic-trending {
+  width: 1.9rem;
+  height: 2.4rem;
+  background-size: 1.9rem 2.4rem;
+}
+
+.ic-drawing {
+  width: 2.2rem;
+  height: 2.2rem;
+  background-size: 2.2rem 2.2rem;
+}
+
+.ic-trash-can {
```

Comment:
There's a few classes that don't appear to be in use here. Can we remove them along with the SVG images that aren't used?